chain/stmgr/stmgr.go

```diff
@@ -54,6 +56,48 @@ type migration struct {
 	upgrade       MigrationFunc
 	preMigrations []PreMigration
 	cache         *nv16.MemMigrationCache
+	resultCache   *resultCache
+}
+
+type resultCache struct {
+	ds        dstore.Batching
+	keyPrefix string
+}
+
+func (m *resultCache) Result(ctx context.Context, root cid.Cid) (cid.Cid, bool, error) {
```

Comment:
super-nit: call this `Get`? When I see `Result`, I'm not sure if we're putting the result or getting it.